### PR TITLE
Add static limit values to LAD tables

### LINKED ISSUE #5051: Add static limit values to LAD tables
**Is your feature request related to a problem? Please describe.**
In the case where static limit values are defined by a limit provider for a given telemetry point, our LAD tables should display those limit values. Currently static limit values are rendered as a limit line in plots, this would bring the same concept to LAD tables.

**Describe the solution you'd like**
Telemetry limit values should be displayed as additional columns in the LAD table

**Additional context**
There is a broader desire to bring all of the customizability of telemetry tables to LAD tables. If considered as part of that change, the new limit columns could be optionally shown or hidden by the user.

### PR DESCRIPTION
**Pull request by scottbell**
Closes #5051 

### Describe your changes:

- Added limits to table columns
- Simplify how table headers are calculated for inspector. Allow limits to be removed

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/tests/functional/plugins/lad/lad.e2e.spec.js [hunk 1]
```diff
@@ -120,6 +164,56 @@ test.describe('Testing LAD table configuration', () => {
...
+    await expect(page.getByRole('cell', { name: 'SEVERE' })).toBeVisible();
+
+    // save and reload and verify they columns are still hidden
+    await page.locator('button[title="Save"]').click();
+    await page.locator('text=Save and Finish Editing').click();
+
+    // Remove Sin Wave Generator
+    openObjectTreeContextMenu(page, sineWaveObject.url);
+    await page.locator('li[role="menuitem"]:has-text("Remove")').click();
+    await page.locator('button:has-text("OK")').click();
```
Comment (ozyx) [inline e2e/tests/functional/plugins/lad/lad.e2e.spec.js:204]:
```suggestion
  await page.getByRole('menuitem', { name: /Remove/ }).click();
  await page.getByRole('button', { name: 'OK' }).click();
```

**Review by ozyx — CHANGES_REQUESTED** [review]
Looking great! Just some minor suggestions regarding some e2e locators. Let's also sync tomorrow and get this feature-tested against VIPER

File: e2e/tests/functional/plugins/lad/lad.e2e.spec.js [hunk 2]
```diff
@@ -120,6 +164,56 @@ test.describe('Testing LAD table configuration', () => {
...
+    // Create Sine Wave Generator
+    await createDomainObjectWithDefaults(page, {
+      type: 'Event Message Generator',
+      parent: ladTable.uuid
+    });
+
+    await page.goto(ladTable.url);
+
+    // Edit LAD table
+    await page.locator('[title="Edit"]').click();
```
[the remark this answers is not in the document]
Comment (scottbell) [reply e2e/tests/functional/plugins/lad/lad.e2e.spec.js]: Note, will need to change this to `page.getByLabel('Edit')` as `page.getByTitle('Edit')` resolves to both "Unlock Editing" and "Edit"

**Review by ozyx — APPROVED** [review]
LGTM

**Pull request merged**